config.go

```diff
@@ -91,6 +95,8 @@ func init() {
 	flag.BoolVar(&options.dryRun, "dryrun", false, "perform a dry run, printing the content to screen")
 	flag.BoolVar(&options.skipTLSVerify, "tls-skip-verify", false, "whether to check and verify the vault service certificate")
 	flag.StringVar(&options.vaultCaFile, "ca-cert", "", "the path to the file container the CA used to verify the vault service")
+	flag.StringVar(&options.vaultClientCertificate, "client-cert", "", "the path to the file container that vault will use to verify the client")
+	flag.StringVar(&options.vaultClientPrivateKey, "client-key", "", "the path to the file container that vault will use to identify the client")
```

Comment:
Do you know what a _File container_ is?

Reply:
Good point - I assumed it was just a file...